plugin/src/main/java/play/soap/plugin/Configuration.java

```diff
@@ -44,4 +48,25 @@ public static boolean exists(String name) {
       return findByName(name) != null;
     }
   }
+
+  public enum Target {
+    PLAY("play");
+
+    final String name;
+
+    Target(String name) {
+      this.name = name;
+    }
+
+    public static Target findByName(String name) {
+      return Arrays.stream(values())
+          .filter(lang -> lang.name.equals(name))
```

Comment:
I think we can compare values without case sensitive. The same behaviour can be for `LANG` parameter.

```suggestion
          .filter(target -> target.name.equalsIgnoreCase(name))
```